treeherder/etl/buildapi.py

```diff
@@ -174,6 +174,16 @@ def transform(self, data, filter_to_project=None, filter_to_revision=None,
             else:
                 log_reference = []
 
+            # add structured logs to the list of log references
+            if 'blobber_files' in prop:
+                blobber_files = json.loads(prop['blobber_files'])
+                for bf, url in blobber_files.items():
+                    if bf.endswith('_raw.log'):
+                        log_reference.append({
+                            'url': url,
+                            'name': 'structured-raw'
```

Comment:
agreed.  I went with ``mozlog_json`` and the api will have ``buildbot_text``.  Another bug will cover converting storage of the artifact from ``builds-4h`` to ``buildbot_text``